cluster/src/dunit/scala/io/snappydata/cluster/SplitSnappyClusterDUnitTest.scala

```diff
@@ -342,8 +342,7 @@ object SplitSnappyClusterDUnitTest
       prop: Properties,
       locatorClientPort: Int): Unit = {
 
-    val snc: SnappyContext = getSnappyContextForConnector(locatorPort,
-      locatorClientPort)
+    val snc: SnappyContext = getSnappyContextForConnector(locatorClientPort)
```

Comment:
Yes, thought so but I thought this would have removed after @dshirish removed his changes of old code. Ok.